prow/gerrit/adapter/adapter.go

```diff
@@ -427,6 +427,19 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 		logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for gerrit change to be processed.")
 	}()
 
+	if change.Revisions != nil {
```

Comment:
I think this change would also deserve a test case.

Reply:
Done.